x-pack/docs/en/security/auditing/output-index.asciidoc

```diff
@@ -47,3 +47,9 @@ NOTE: Audit events are batched for indexing so there is a lag before
 events appear in the index. You can control how frequently batches of
 events are pushed to the index by setting
 `xpack.security.audit.index.flush_interval` in `elasticsearch.yml`.
+The value should adhere to the `TimeValue` datatype, the default is 1s (1 second).
```

Comment:
Can we remove this sentence, for two reasons: (1) we don't really document `TimeValue` in a way that users of these docs would reference, and (2) the default for this setting is already documented [elsewhere](https://github.com/elastic/elasticsearch/blame/64e5c254c1b5ed13217b0212abde53482bfe3e13/docs/reference/settings/audit-settings.asciidoc#L124)

Reply:
ok, i will change